treeherder/etl/management/commands/ingest.py

```diff
@@ -359,64 +419,73 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-        typeOfIngestion = options["ingestion_type"][0]
```

Comment:
Would you mind look at how to make the diff for this function change less? It will make reviewing it easier.